data/json/monsters/mammal.json

```
@@ -2376,6 +2376,39 @@
     "petfood": { "food": [ "CATTLEFOOD" ], "feed": "The %s seems to like you!  It lets you pat its head and seems friendly." },
     "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "PET_WONT_FOLLOW", "MILKABLE" ]
   },
+  {
+    "id": "mon_goat_kid",
+    "type": "MONSTER",
+    "name": { "str": "kid" },
```

```suggestion
    "name": { "ctxt": "goat", "str": "kid" },
```
Otherwise, the player might feel remorse when killing one of these.